includes/Pager.php

```diff
@@ -309,10 +316,48 @@ function reallyDoQuery( $offset, $limit, $descending ) {
 		}
 		$options['LIMIT'] = intval( $limit );
 		$res = $this->mDb->select( $tables, $fields, $conds, $fname, $options, $join_conds );
+
 		return new ResultWrapper( $this->mDb, $res );
 	}
 
 	/**
+	 * Fetch user names from ExternalSharedDB.
+	 * @see SUS-2990
+	 * @return array
+	 */
+	protected function fetchUsersForHistoryPage() {
```

Comment:
This shouldn't be in `IndexPager` class.

Reply:
Any proposals? :)

Reply:
We could move this logic to `HistoryPager` class.